apiserver/backrestservice/backrestimpl.go

```diff
@@ -593,3 +623,18 @@ func createRestoreWorkflowTask(clusterName, ns string) (string, error) {
 	}
 	return spec.Parameters[crv1.PgtaskWorkflowID], err
 }
+
+// clusterNamesToPGClusterList takes a list of cluster names as specified by a slice of
+// strings containing cluster names and then returns a PgclusterList containing Pgcluster's
+// corresponding to those names.
+func clusterNamesToPGClusterList(clusterNames []string, namespace string) (crv1.PgclusterList,
+	error) {
+	selector := fmt.Sprintf("%s in(%s)", config.LABEL_NAME, strings.Join(clusterNames, ","))
```

Comment:
I think just a `%s` for the `clusterNames` is fine, per [docs](https://golang.org/pkg/fmt/) and it saves us from having to do any extra computational work with the join.

Reply:
I need to ensure the values are comma separated for the proper Kube selector syntax, e.g.:
```bash
kubectl get pgcluster --selector="name in(mycluster1, mycluster2, mycluster3)"
```

Reply:
Oh I thought this was a debug statement, sorry. Ignore me.